<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 40, 49);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
}

PlotWidget {
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 12pt &quot;Chakra Petch Medium&quot;;
}

QLabel, QCheckBox {
	background-color: rgb(57, 62, 70);
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 &quot;Chakra Petch Medium&quot;;
}

QFrame#command_control_panel, QFrame#XBEE_port_control_panel, QFrame#sim_data_control_panel  {
	border: 1px solid rgb(238, 238, 238);
}


*{
	color: rgb(238, 238, 238);
	border-radius: 15px;
	font: 75 &quot;Chakra Petch Medium&quot;;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="left_side_layout" stretch="2,1,4,4,4">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="TeamHeader">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="team_logo">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Chakra Petch Medium</family>
             <pointsize>23</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 211, 105)</string>
           </property>
           <property name="text">
            <string>CanBee #1008</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Chakra Petch Medium</family>
          <pointsize>26</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: none; color: rgb(255, 211, 105)</string>
        </property>
        <property name="text">
         <string>Control Panels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="command_control_panel">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Chakra Petch Medium</family>
               <pointsize>18</pointsize>
               <weight>9</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: none;
color: rgb(255, 211, 105);</string>
             </property>
             <property name="text">
              <string>Command Control Panel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="command_line">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Chakra Petch Medium</family>
               <pointsize>12</pointsize>
               <weight>9</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">	background-color: rgb(57, 62, 70);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Chakra Petch Medium</family>
               <pointsize>12</pointsize>
               <weight>9</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">	background-color: rgb(57, 62, 70);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">text-align: center;
background-color: rgb(67, 207, 153);
color: rgb(44, 51, 51)
</string>
           </property>
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="XBEE_port_control_panel">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Chakra Petch Medium</family>
             <pointsize>18</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background: none;
color: rgb(255, 211, 105);</string>
           </property>
           <property name="text">
            <string>XBEE Port Control Panel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">text-align: left;
background-color: rgb(15, 98, 146);</string>
           </property>
           <property name="text">
            <string>Check XBEE Ports</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Chakra Petch Medium</family>
             <pointsize>12</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;port returns here&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">text-align: center;
background-color: rgb(67, 207, 153);
color: rgb(44, 51, 51)
</string>
           </property>
           <property name="text">
            <string>Send XBEE Port</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="sim_data_control_panel">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Chakra Petch Medium</family>
             <pointsize>18</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background: none;
color: rgb(255, 211, 105)</string>
           </property>
           <property name="text">
            <string>Sim Data Control Panel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">text-align: left;
background-color: rgb(15, 98, 146);</string>
           </property>
           <property name="text">
            <string>Browse Sim Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Chakra Petch Medium</family>
             <pointsize>12</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;sim data returns here&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">text-align: center;
background-color: rgb(67, 207, 153);
color: rgb(44, 51, 51)
</string>
           </property>
           <property name="text">
            <string>Send Sim Data</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="right_side_layout" stretch="0,5,3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="telemetry_state">
        <property name="spacing">
         <number>26</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Chakra Petch Medium</family>
            <pointsize>18</pointsize>
            <weight>9</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: none; color: rgb(255, 211, 105)</string>
          </property>
          <property name="text">
           <string>Telemetry Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Chakra Petch Medium</family>
            <pointsize>12</pointsize>
            <weight>9</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: none; color:rgb(67, 207, 153);</string>
          </property>
          <property name="text">
           <string>State:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="graph_layout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="3,3,2">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="PlotWidget" name="widget_PRESSURE" native="true"/>
          </item>
          <item>
           <widget class="PlotWidget" name="widget_TEMPERATURE" native="true"/>
          </item>
          <item>
           <widget class="QLabel" name="label_GPS_SATS">
            <property name="font">
             <font>
              <family>Chakra Petch Medium</family>
              <pointsize>12</pointsize>
              <weight>9</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(34, 40, 49);</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;GPS Sats: 0&lt;/p&gt;&lt;p&gt;GPS Latitude: XX.XXXX&lt;/p&gt;&lt;p&gt;GPS Longitude: YY.YYYY&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="PlotWidget" name="widget_ALTITUDE" native="true"/>
          </item>
          <item>
           <widget class="PlotWidget" name="widget_GPS_ALTITUDE" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="PlotWidget" name="widget_VOLTAGE" native="true"/>
          </item>
          <item>
           <widget class="PlotWidget" name="widget_TILT_X" native="true"/>
          </item>
          <item>
           <widget class="PlotWidget" name="widget_TILT_Y" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(238, 238, 238)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
